### PR TITLE
Add pod-spec-get command

### PR DESCRIPTION
**Pull request by wallyworld**
## Description of change

Add new pod-spec-get hook command for k8s charms to use.

## QA steps

deploy k8s charm
juju exec --unit foo/0 pod-spec-get

## Documentation changes

k8s charm doc will need refreshing

### CODE REVIEW

**Comment by wallyworld** [comment]
[pod-spec-get](https://trello.com/c/t5zkgdZY/981-pod-spec-get)

File: worker/uniter/runner/jujuc/pod-spec-get_test.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+Details:
+Gets configuration data used for a pod.
+`[1:]
+
+	c.Assert(bufferString(ctx.Stdout), gc.Equals, expectedHelp)
+	c.Assert(bufferString(ctx.Stderr), gc.Equals, "")
+}
+
+func (s *PodSpecGetSuite) TestContainerSpecSet(c *gc.C) {
```
Comment (ycliuhw) [inline worker/uniter/runner/jujuc/pod-spec-get_test.go]: `TestPodSpecGet`?

Comment (wallyworld) [reply worker/uniter/runner/jujuc/pod-spec-get_test.go]: derp

**Comment by wallyworld** [comment]
$$merge$$

**Pull request merged**